### PR TITLE
Update Polish

### PR DESCRIPTION
**Pull request by circl-lastname**

### CODE REVIEW

File: texts/index.js [hunk 1]
```diff
@@ -280,9 +280,9 @@ texts.push([
 	"We hebben een blog waar we maandelijks nieuwe gedetaileerde rapporten publiceren met het laatst geboekte voortgang in ons werk. Neem een bezoekje om te kijken wat nieuw is!" // 35
 ])
 
-// Polish (Translation provided by MAX0342)
+// Polish (Translation provided by circl)
```
Comment (VelocityRa) [inline texts/index.js]: keep the previous name too

Comment (IllusionMan1212) [reply texts/index.js]: beat me to it :p @VelocityRa

**Comment by VelocityRa** [comment]
I'd auto commit the change but you have disabled changes from repo maintainers, so you'll have to push another commit fixing the above issue yourself.

**Comment by circl-lastname** [comment]
Thanks!

**Pull request merged**